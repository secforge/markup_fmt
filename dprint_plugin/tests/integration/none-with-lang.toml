# Test that "none" mode ignores lang attribute
[vue_custom_block]
docs = "none"
