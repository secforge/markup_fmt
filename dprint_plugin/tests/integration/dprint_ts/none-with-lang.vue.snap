---
source: dprint_plugin/tests/integration.rs
assertion_line: 123
---
<template>
  <div>Test none mode with lang attribute</div>
</template>

<!-- docs block with "none" mode should preserve content even with lang="html" -->
<docs lang="html">
  <p>This    content    should    be    preserved    exactly</p>
  <div>
    <span>No    formatting    should    occur</span>
  </div>
</docs>

<!-- docs block with "none" mode and lang="json" -->
<docs lang="json">
{
"key":    "value",
  "another": "value"
}
</docs>
